### PR TITLE
Improve browser experience

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions,_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses [#458](https://github.com/MetaMask/mobile-planning/issues/458)

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: patches/react-native-webview+11.13.0.patch [hunk 1]
```diff
@@ -1,3 +1,4161 @@
...
+new file mode 100644
+index 0000000..74ec290
+Binary files /dev/null and b/node_modules/react-native-webview/android/build/intermediates/compile_r_class_jar/debug/R.jar differ
+diff --git a/node_modules/react-native-webview/android/build/intermediates/compile_symbol_list/debug/R.txt b/node_modules/react-native-webview/android/build/intermediates/compile_symbol_list/debug/R.txt
+new file mode 100644
+index 0000000..3a5f626
+--- /dev/null
++++ b/node_modules/react-native-webview/android/build/intermediates/compile_symbol_list/debug/R.txt
+@@ -0,0 +1,1857 @@
++int anim abc_fade_in 0x0
```
Comment (sethkfman) [inline patches/react-native-webview+11.13.0.patch:98]: Can you run a `build clean` then retry the patch-package? These files are build files we don't want.

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
 	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 	<uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 	<uses-permission android:name="android.permission.CAMERA" />
+	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```
Comment (NicholasEllul) [inline android/app/src/main/AndroidManifest.xml:10]: Reading the `write_external_storage` permission in https://developer.android.com/reference/android/Manifest.permission#WRITE_EXTERNAL_STORAGE, it notes: 

> Starting in API level 19, this permission is not required to read/write files in your application-specific directories returned by [Context.getExternalFilesDir(String)](https://developer.android.com/reference/android/content/Context#getExternalFilesDir(java.lang.String)) and [Context.getExternalCacheDir()](https://developer.android.com/reference/android/content/Context#getExternalCacheDir()).

and 

> Allows access to external storage. Protection level: dangerous

So I have 2 questions:

1. If a user still declines storage permissions, is there any way that the browser may access `getExternalFilesDir` which doesn't require this permission?
2. Using this gives more user control, but when the permission is granted it, now lets the MetaMask now have access to more external storage devices. Is there any way around having to do this?

Comment (blackdevelopa) [reply android/app/src/main/AndroidManifest.xml:10]: Hey Nic, great questions. Moving this conversation to [144](https://github.com/MetaMask/pm-security/issues/144)

**Comment by blackdevelopa** [comment]
Closing this PR for [5238](https://github.com/MetaMask/metamask-mobile/pull/5238)

**Pull request closed without merging**